README.md

```diff
@@ -0,0 +1,14 @@
+## Purpose:
+To send notification messages to various discord channels, reminding people that there is an event today.
+
+## Tools Used
+- Node
```

Comment:
We should probably include a link to the discord documentation so new folks know where to look for API's.

Reply:
Great point, done!